environments/.gitignore

```diff
@@ -6,11 +6,12 @@
 # Auto-generated by amplify, but with `**` added to cover all environments
 **/amplify/\#current-cloud-backend
 **/amplify/.config/local-*
-**/amplify/logs
-**/amplify/mock-data
 **/amplify/backend/amplify-meta.json
 **/amplify/backend/awscloudformation
 **/amplify/backend/.temp
+**/amplify/hooks
```

Comment:
QQ: what is under the hooks folder? Shouldn't content in this file be auto-generated?

Reply:
There is a `README` that escapes our current `.gitignore` patterns

Reply:
Ouch..